docs/windows-requirements.md

```diff
@@ -0,0 +1,23 @@
+# Windows
+
+## Requirements to build native modules
+
+If you want to build native modules, make sure that the following tools are installed on your system.
+
+- [Node 14](https://nodejs.org)
+- [Python 3](https://www.python.org/downloads/)
+- [Strawberry Perl](https://strawberryperl.com/)
+- [Rust](https://rustup.rs/)
+- [Visual Studio](https://visualstudio.microsoft.com/downloads/) with the following packages
+  - MSVC VS 2019 C++
+  - Windows 10 SDK
+  - C++ Clang tools
```

Comment:
I know you installed the full IDE, but I think it's possible to get away with just the build tools, so perhaps we can suggest that...?

```suggestion
- [Build Tools for Visual Studio 2019](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019) with the following configuration:
  - On the Workloads tab:
    - Desktop & Mobile -> C++ build tools
  - On the Individual components tab:
    - MSVC VS 2019 C++ build tools
    - Windows 10 SDK (latest version available)
    - C++ CMake tools for Windows
```

Reply:
Hmm, I'm not entirely sure. Can you install the build tools without Visual Studio itself?
Let me try to play around with the system for a bit to figure this one out

Reply:
Yes, that's my understanding at least, which [this page](https://docs.microsoft.com/en-us/visualstudio/msbuild/walkthrough-using-msbuild?view=vs-2019) seems to agree with:

> To install MSBuild on a system that doesn't have Visual Studio, go to Visual Studio downloads and scroll down to All Downloads, then expand Tools for Visual Studio 2019. Install Build Tools for Visual Studio 2019...